docs/develop/contracts/js/quickstart.md

```diff
@@ -1,66 +1,86 @@
+| Synchronous Cross-Contract Calls           | ❌            | ✅          |
+| Call methods on any smart contract on NEAR | ✅            | ❌          |
+| Standards Support                          | ✅            | Not in v1.0 |
+| Function Call Access Key Support           | ✅            | Not in v1.0 |
 
-| Areas                     | WebAssembly                                    | Enclave                                                                                                                                                     |
-|---------------------------|--------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Can interact with any smart contract on NEAR                     |✅|❌|
-| Synchronous Cross-Contract Calls                                 |❌|✅|
-| Standards Support                                                |✅|Not in v1.0|
-| Function Call Access Key Support                                 |✅|Not in v1.0|
+In addition to the ability to develop in a language you are already familiar with, the `JSVM` operates in a synchronous environment which provides an easy way to create smart contracts that interact with other contracts know as performing "cross-contract calls.
 
-The JavaScript Enclave is a very powerful tool to help you kickstart your smart contract programming journey. Writing contracts in JavaScript is much easier than learning Rust, and thanks to the jsvm's ability to handle cross-contract calls synchronously can greatly help with people's understanding of how contracts can work.
+:::warning Heads up
+JavaScript smart contract development is not recommended for financial use cases as it is still very new to the NEAR ecosystem.
+:::
 
-## Quickstart {#quickstart}
+---
 
-In this quick-start guide, you'll learn the basics of setting up a new JavaScript smart contract on the enclave that stores and retrieves a greeting message. You'll then create a simple web-based frontend that displays the greeting and allows you to change it.
+## Requirements
 
-### Prerequisites
+- [Node.js & npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [near-cli](https://docs.near.org/docs/tools/near-cli) `v3.1.1` or greater
 
-In order to successfully complete this quickstart guide, you'll need to have a few things installed:
-- [Node.js](https://nodejs.org/en/about/) and [npm](https://www.npmjs.com/):
 ```bash
-curl -sL https://deb.nodesource.com/setup_17.x | sudo -E bash -  
-sudo apt install build-essential nodejs
-PATH="$PATH"
-```
-Ensure that they are both installed by running a version check:
-```
-node -v
-npm -v
+npm i -g near-cli
 ```
 
-It's important to have the **newest** version of the [NEAR-CLI](https://docs.near.org/docs/tools/near-cli) installed such that you can make use of the JavaScript features. To install or update, run: 
```

Comment:
I wouldn't remove this because people that already have near-cli won't be able to go through the tutorial.

Reply:
we could render this paragraph as a "tip", using:
```
:::tip
It's important to....
:::
```

Reply:
I did put that `v3.1.1` or greater is a requirement in favor of this, but will add an additional warning.